src/tests/ftest/pool/info_tests.py

```diff
@@ -47,8 +47,8 @@ def test_pool_info_query(self):
         :avocado: tags=all,tiny,pr,pool,smoke,infotest
         """
         # Get the test params
-        self.pool = TestPool(self.context, self.log,
-                             dmg_command=self.get_dmg_command())
+        self.pool = TestPool(self.context, self.get_dmg_command(),
+                             log=self.log)
```

Comment:
The `log` argument is defunct for TestPool.create():
```suggestion
        self.pool = TestPool(self.context, self.get_dmg_command())
```